CMakeLists.txt

```diff
@@ -31,7 +31,8 @@ option(ENABLE_DILITHIUM "Enable Dilithium signatures in the EVP API" OFF)
 option(DISABLE_PERL "Disable Perl for AWS-LC" OFF)
 option(DISABLE_GO "Disable Go for AWS-LC" OFF)
 option(ENABLE_FIPS_ENTROPY_CPU_JITTER "Enable FIPS entropy source: CPU Jitter" OFF)
-option(ENABLE_DATA_INDEPENDENT_TIMING_AARCH64 "Enable Data-Independent Timing (DIT) flag on Arm64" OFF)
+option(ENABLE_DATA_INDEPENDENT_TIMING_AARCH64 "Enable automatic setting/resetting Data-Independent Timing
```

Comment:
* Should we also change the name here to `AUTO_ENABLE_DATA_INDEPENDENT_TIMING`?
* Should we make this build flag platform-agnostic, so we can re-use it for any platform (not just AARCH64) that requires special handling for DIT?

Reply:
regarding the second bullet point, if we do that we need to be clear that this is only available on aarch64, either through documentation or perhaps aborting if it's set to ON on x86.

Reply:
Isn't the current assumption on x86-64 (and other platforms) that the *timing* of our encryption operations are (or are expected to be) *data-independent*? So enabling it elsewhere is essentially a no-op for the build.

Reply:
> Isn't the current assumption on x86-64 (and other platforms) ....

Oh, I wasn't aware of this. If so, agreed.